Rakefile

```diff
@@ -15,6 +17,48 @@ task :default => [:spec]
 
 Rake::Task["db:structure:dump"].clear unless Rails.env.development?
 
+def connect_sequel
+  if ENV['LOGS_DATABASE_URL'].nil?
+    raise 'No logs database URL set, you must set LOGS_DATABASE_URL env var'
```

Comment:
The logs database isn't required everywhere - can we make this a warning instead?